src/video_core/CMakeLists.txt

```
@@ -250,6 +269,12 @@ create_target_directory_groups(video_core)
 target_link_libraries(video_core PUBLIC common core)
 target_link_libraries(video_core PRIVATE glad xbyak)
 
+if (MSVC)
+    target_link_libraries(video_core PUBLIC ffmpeg::ffmpeg)
+else()
+    target_link_libraries(video_core PRIVATE ${FFMPEG_LIBAVCODEC} ${FFMPEG_LIBAVUTIL} ${FFMPEG_LIBSWSCALE})
```

```suggestion
    target_include_directories(video_core PRIVATE ${FFMPEG_INCLUDE_DIR})
    target_link_libraries(video_core PRIVATE ${FFMPEG_LIBRARIES})
```
Uses the variables defined in FindFFmpeg.cmake, as well as properly telling the compiler where the headers are.